docs/resources/Channel.md

```diff
@@ -893,6 +893,7 @@ Files must be attached using a `multipart/form-data` body as described in [Uploa
 | Field                 | Type                                                                                              | Description                                                                                                                                                                 |
 | --------------------- | ------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | content?\*            | string                                                                                            | Message contents (up to 2000 characters)                                                                                                                                    |
+| nonce?                | integer or string                                                                                 | a nonce that can be used for message roundtrips with the gateway (up to 25 characters)                                                                                      |
```

Comment:
It can be added back, but I'd suggest aligning the language closer with what's currently on the [message object](https://discord.com/developers/docs/resources/channel#message-object).
```suggestion
| nonce?                | integer or string                                                                                 | Can be used to verify a message was sent                                                                                      |
```